### PR TITLE
added topic creation options

### PR DESCRIPTION
**Pull request by filiptepper**
added replication factor and partitions to KAFKA_CREATE_TOPICS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,10 +45,12 @@ added in ```docker-compose.yml```.
 Here is an example snippet from ```docker-compose.yml```:
 
         environment:
-          KAFKA_CREATE_TOPICS: "Topic1,Topic2"
+          KAFKA_CREATE_TOPICS: "Topic1,1,3:Topic2,1,1"
+
+```Topic 1``` will have 1 partition and 3 replicas, ```Topic 2``` will have 1 partition and 1 replica.
 
 NOTE: All topics will be created with a single partition and a single replica
```
Comment (wurstmeister) [inline README.md]: this note is not required anymore

File: README.md [hunk 2]
```diff
@@ -45,10 +45,12 @@ added in ```docker-compose.yml```.
 Here is an example snippet from ```docker-compose.yml```:
 
         environment:
-          KAFKA_CREATE_TOPICS: "Topic1,Topic2"
+          KAFKA_CREATE_TOPICS: "Topic1,1,3:Topic2,1,1"
```
Comment (wurstmeister) [inline README.md]: i think it would be easier to read it you swap the ```,``` and ```:``` i.e. ```firsttopic:1:3,secondtopic:1:1```

**Comment by filiptepper** [comment]
Thanks you for your suggestions @wurstmeister. Here are the changes.

**Comment by wurstmeister** [comment]
Thank you

**Pull request merged**